app/build.gradle

```diff
@@ -3,12 +3,17 @@ plugins {
     id 'kotlin-android'
 }
 
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+apply plugin: 'kotlin-kapt'
+
 android {
     compileSdkVersion 30
     buildToolsVersion "30.0.3"
 
     defaultConfig {
-        applicationId "com.example.internshipandroidpart3"
+        applicationId "com.example.androidinternshippart3"
```

Comment:
Package should be com.onix.internship.survay

Reply:
But **com.example.androidinternshippart3** is not same with **com.onix.internship.survay**